addons/l10n_in_ewaybill_stock/report/ewaybill_report.xml

```diff
@@ -15,74 +15,90 @@
                                   font-size: 14px;
                                 }
                             </style>
-
+                            <t t-set="challan_states" t-value="('challan', 'pending')"/>
```

Comment:
I would prefer to define `ewaybill_states` since pending doesn't belongs to neither challan/e waybill, but works fine